cardano-node/src/Cardano/Tracing/Tracers.hs

```diff
@@ -338,8 +340,15 @@ mkTracers traceOptions tracer = do
         = annotateSeverity $ filterSeverity (pure . const (tracingSeverity $ hasConsensusTraceFlag Consensus.forgeTracer))
           $ teeForge forgeTracers (tracingFormatting $ hasConsensusTraceFlag Consensus.forgeTracer) tracingVerbosity
           $ addName "Forge" tracer
+      , Consensus.blockchainTimeTracer
+        = Tracer $ \ev -> traceWith (toLogObject tracer) (readableTraceBlockchainTimeEvent ev)
       }
 
+    readableTraceBlockchainTimeEvent :: TraceBlockchainTimeEvent -> Text
+    readableTraceBlockchainTimeEvent ev = case ev of
```

Comment:
could be a transformer using "annotateSeverity" and "toLogObject'", then we need to add a the orphan instances to make it work. could do it in about an hour after dinner.

Reply:
This is not a regular trace, it is a single message on startup, just like
https://github.com/input-output-hk/cardano-node/blob/cef513628f35180aca9e8be1ff20b9fc11cb9267/cardano-node/src/Cardano/Node/Run.hs#L161